src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/entities/ccd/callback/Callback.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.iahearingsapi.domain.entities.ccd.callback;
```

Comment:
@rajeshthuraiyur and @sibuckley-v1, are we going with the approach of callbacks (PreSubmitCallBack, PostSubmitCallBack, etc...)? We might not need these. @Dbarnes-v1 is aware of this as well.

Reply:
Sticking with callbacks until I have a business justification/requirements.